strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/yaml/configuration/repository/MavenRepositoryConfigurationData.java

```diff
@@ -63,5 +68,9 @@ public String getMetadataExpirationStrategy()
         return metadataExpirationStrategy;
     }
 
+    @Override
+    public Set<String> getDigestAlgorithmSet() {
+        return digestAlgorithmSet;
+    }
```

Comment:
Cosmetic: Code style.

```suggestion
    public Set<String> getDigestAlgorithmSet()
    {
        return digestAlgorithmSet;
    }
```